### DISCUSSION TITLE
是否忘记了修改output参数？

### DISCUSSION THREAD

**Discussion by slideslide** [GaiZhenbiao/ChuanhuChatGPT#988] [Q&A]
4 turbo 1106版本遇到了如下问题
☹️发生了错误：{ "error": { "message": "max_tokens is too large: 5566. This model supports at most 4096 completion tokens, whereas you provided 5566.", "type": "invalid_request_error", "param": "max_tokens", "code": null }}
理论上这个版本应该有更高的可输入量

**Comment by GaiZhenbiao** [comment]
确实应该有更高的token限制。你是在什么情况下遇到的？

**Reply by slideslide** [reply]
单纯就是用得猛了点，非要说什么情况就是正常情况使用，以及新模型token扩充了自动总结的限制还是那么点。

**Reply by GaiZhenbiao** [reply]
这个是OpenAI报的错误，不是川虎Chat的错误。所以我也很奇怪为什么会说token超限了🤔

**Reply by slideslide** [reply]
很怪，怎会如此？以及自动总结的触发也是很频繁，128000才用到4000多就给自动总结掉了，尽管能接上但是还是会遗忘不少细节。

**Reply by GaiZhenbiao** [reply]
我测试下来，是没有问题的。你可以试着拉取一下最新版本的代码。
<img width="393" alt="image" src="https://github.com/GaiZhenbiao/ChuanhuChatGPT/assets/51039745/b386843c-ee3a-46d3-b4a4-e1e9834c6229">

**Reply by slideslide** [reply]
感谢，似乎确实是老版本的问题，拉取最新版本后确实解决了，不过自动总结还是太早了。

**Reply by slideslide** [reply]
不知道和model_token_mapping是否相关
